host-controller/src/main/java/org/jboss/as/domain/controller/resources/DomainDeploymentResourceDefinition.java

```diff
@@ -125,4 +131,26 @@ protected void registerAddOperation(ManagementResourceRegistration registration,
         registration.registerOperationHandler(addDefinition, handler);
     }
 
+    static class ServerGroupDomainDeploymentResourceDefinition extends DomainDeploymentResourceDefinition {
+         ServerGroupDomainDeploymentResourceDefinition(DeploymentResourceParent parent, OperationDefinition addDefinition,
+                 OperationStepHandler addHandler, OperationStepHandler removeHandler,
+                 OperationStepHandler explodeDeploymentHandler, OperationStepHandler explodedDeploymentAddContentHandler,
+                 OperationStepHandler explodedDeploymentRemoveContentHandler,
+                 OperationStepHandler explodedDeploymentReadContentHandler,
+                 OperationStepHandler explodedDeploymentBrowseContentHandler) {
+             super(parent, addDefinition, addHandler, removeHandler, explodeDeploymentHandler,
+                     explodedDeploymentAddContentHandler, explodedDeploymentRemoveContentHandler,
+                     explodedDeploymentReadContentHandler, explodedDeploymentBrowseContentHandler);
+         }
+
+         @Override
+         public void extractedManaged(OperationContext context, ModelNode operation) {
+              String name = PathAddress.pathAddress(operation.get(ModelDescriptionConstants.OP_ADDR)).getLastElement().getValue();
```

Comment:
Shouldn't you use context.getCurrentAddressValue() instead ?

Reply:
switched `context.getCurrentAddressValue()` instead